test/library/standard/IO/open/checkPermissions.chpl

```diff
@@ -0,0 +1,19 @@
+use IO, OS.POSIX, FileSystem;
+
+var filename = "readOnly.txt";
+
+var w = openwriter(filename);
+w.writeln("something");
+w.close();
+
+chmod(filename, 0o0400); // set it to read only now that it has contents
```

Comment:
I'm worried this might not pass nightly testing on the windows VM. Do you know if this will work on Windows too?

Reply:
No, I'm not too worried about it if the other test passes there as well. Thanks!

Reply:
Looks like I should have checked it, d'oh!

Reply:
Shoot darn... I wonder why that other test passes on cygwin then?

Reply:
I think it might be that running as a windows service suppresses permission errors.  There's a different test that looked at generating an error specifically and that is suppressed when we're run as a windows service but has specific cygwin output for running locally.

Reply:
So I think we can change the permissions appropriately, but the checking is weird